base/dict.jl

```diff
@@ -351,7 +351,7 @@ function ht_keyindex2!(h::Dict{K,V}, key) where V where K
     return ht_keyindex2!(h, key)
 end
 
-@propagate_inbounds function _setindex!(h::Dict, v, key, index)
+@propagate_inbounds @inline function _setindex!(h::Dict, v, key, index)
```

Comment:
Doesn't `@propagate_inbounds` already do inlining?

Reply:
```suggestion
@propagate_inbounds function _setindex!(h::Dict, v, key, index)
```
Indeed, that is what the docs say.